cli/internal/commands/authorize_cluster/generator.go

```diff
@@ -272,6 +272,41 @@ func (o *GeneratorOptions) clientInfo(ctx context.Context, ctrl *config.Controll
 	return o.Config.RegisterClient(ctx, client)
 }
 
+// clusterClientInformation reads the registered client from the cluster, allowing it to be re-used.
+func (o *GeneratorOptions) clusterClientInformation(ctx context.Context, ctrl *config.Controller) *registration.ClientInformationResponse {
+	cmd, err := o.Config.Kubectl(ctx, "get", "secret", "--namespace", ctrl.Namespace, o.Name,
+		"--output", "go-template", "--template",
+		"{{ .data.STORMFORGE_AUTHORIZATION_CLIENT_ID }}/{{.data.STORMFORGE_AUTHORIZATION_CLIENT_SECRET }}")
```

Comment:
unsure if we want to get cute or not, but we could potentially handle the decoding inside the template --
```
kubectl get secret ...   -o go-template --template '
{{- $id := .data.STORMFORGE_AUTHORIZATION_CLIENT_ID -}}
{{- $secret := .data.STORMFORGE_AUTHORIZATION_CLIENT_SECRET -}}
{{- if and $id $secret }}{{ $id | base64decode }}/{{ $secret | base64decode }}{{ end -}}
'
```

more just a fyi, not requesting change